_release_notes/armoryspinnaker_v1.14.65.md

```diff
@@ -0,0 +1,926 @@
+---
+layout: post
+title: v1.14.65 Armory Release
+order: -20180716195513
+hidden: false
+---
+
+# 07/16/18 Release Notes
+{:.no_toc}
+
+> Note: If you're experiencing production issues after upgrading Spinnaker, rollback to a [previous working version](http://docs.armory.io/admin-guides/troubleshooting/#i-upgraded-spinnaker-and-it-is-no-longer-responding-how-do-i-rollback) and please report issues to [http://go.armory.io/support](http://go.armory.io/support).
+
+* This is a placeholder for an unordered list that will be replaced with ToC. To exclude a header, add {:.no_toc} after it.
+{:toc}
+
+
+## Known Issues
+There's currently no known issues with this release.
+
+<!--- Example of a problem
+Igor added ..... which does.....
+
+**Symptoms:**
+**Fix:**
+-->
```

Comment:
Should this "Example" be removed if we don't know of any?

Reply:
Nah, let's leave it so we know the format when we do find an issue to report on this version